template/README.mustache

```diff
@@ -247,31 +263,31 @@ To learn more, refer to example located in [src/examples/java/com/fingerprint/ex
 
 ## Webhook signature validation
 This SDK provides utility method for verifying the HMAC signature of the incoming webhook request.
+
+Here is an example how to made it using Spring Boot.
```

Comment:
I'd suggest a different sentence here, such as "Here is an example implementation using Spring Boot"

Reply:
```suggestion
Here is an example implementation using Spring Boot: 
```

Reply:
This suggestion should be applied to `README.md` file as well, given the fact that this file is `.mustache` file.